Packs/AggregatedScripts/Scripts/ExpirePassword/ExpirePassword.py

```diff
@@ -0,0 +1,468 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+
+from typing import Literal, TypedDict
+from collections.abc import Callable
+
+
+class ExpiredPasswordResult(TypedDict):
+    """Represents the result of a user password expiration operation."""
+
+    Result: Literal["Success", "Failed"]
+    Message: str
+    Instance: str
+
+
+class UserData(TypedDict):
+    """Represents the data of a user retrieved from get-user-data."""
+
+    ID: str
+    Username: str
+    Email: str
+    Status: str
+    Brand: str
+    Instance: str
+
+
+# --- Constants ---
+# Success message patterns for different integrations
+AD_NEVER_EXPIRE_CLEARED = 'AD account {username} has cleared "password never expire" attribute. Value is set to False'
```

Comment:
I think it will be better to group this under a dictionary (same for failed)
```
SUCCESS_MESSAGES = {
    "ad_never_expire_cleared": 'AD account {username} has cleared "password never expire" attribute. Value is set to False',
    "ad_password_expired": "Expired password successfully",
    "msgraph_password_reset": "User {username} will be required to change his password.",
    etc etc 
}
```